doc/iterations/2023.12.0/shapes/rotatable_border_node.adoc

```diff
@@ -0,0 +1,45 @@
+= (S) Rotatable border node
+
+== Problem
+
+When using border nodes with image, if the border node is moved around its parent node, the image will keep its orientation in every position.
+Depending on the choice of image, specifiers may want the image to rotate when changing the edge anchor of its parent node.
+
+== Key Result
+
+Specifiers must be able to declare a `imageNodeDescription` as rotatable.
+When an end user moves a rotatable border node image around its parent, the image must always be oriented to the center of this parent.
+
+== Solution
+
+=== Backend
+
+We will add a variable named `isBorderNode` for the node style computation.
```

Comment:
Where this variable will be, the AQL interpreter?

Reply:
Yes, in the variable manager used for conditionalStyle. I added a note about it in the shape.